dashboard/src/containers/DeploymentStatusContainer/DeploymentStatusContainer.ts

```diff
@@ -0,0 +1,23 @@
+import * as _ from "lodash";
+import { connect } from "react-redux";
+
+import DeploymentStatus from "../../components/DeploymentStatus";
+import { hapi } from "../../shared/hapi/release";
+import ResourceRef from "../../shared/ResourceRef";
+import { IStoreState } from "../../shared/types";
+import { filterByResourceRefs } from "../helpers";
+
+interface IDeploymentStatusContainerProps {
+  deployRefs: ResourceRef[];
+  info?: hapi.release.IInfo;
+}
+
+function mapStateToProps({ kube }: IStoreState, props: IDeploymentStatusContainerProps) {
+  const { deployRefs, info } = props;
+  return {
+    deployments: filterByResourceRefs(deployRefs, kube.items),
```

Comment:
we should add a comment here explaining that we rely on DeploymentItems for fetching deployments

Reply:
good shout.